lib/fluent/test/driver/base_owned.rb

```diff
@@ -53,13 +53,26 @@ def initialize(klass, opts: {}, &block)
           @section_name = ''
         end
 
-        def configure(conf, syntax: :v1)
+        def configure(conf)
```

Comment:
this change doesn't impact existing test cases?

Reply:
looks like so. this was introduced in https://github.com/fluent/fluentd/pull/1047 . but I can't get any more information about why this method needs this parameter. so leave it as it is. and I will delete it when fluentd becomes v2. or you think it should be deleted in this PR?

Reply:
There is no problem for me because this is only for test, not for production code.
Failing test is not critical and easy to fix it.